feature-libs/user/account/components/login-form/login-form-component.service.ts

```diff
@@ -22,14 +26,28 @@ import { tap, withLatestFrom } from 'rxjs/operators';
 
 @Injectable()
 export class LoginFormComponentService {
+  protected authConfigService = inject(AuthConfigService);
+  protected isNewAuthFlow = inject(FeatureConfigService).isEnabled(
+    'authorizationCodeFlowByDefault'
+  );
+
   constructor(
     protected auth: AuthService,
     protected globalMessage: GlobalMessageService,
     protected winRef: WindowRef
-  ) {}
+  ) {
+    if (this.isNewAuthFlow) {
+      this.method = 'POST';
+      this.action = this.authConfigService?.getCustomLoginFormEndpoint();
+    }
```

Comment:
According to the previous comment: 
```suggestion
    if (this.featureConfigService.isEnabled('authorizationCodeFlowByDefault')) {
      this.method = 'POST';
      this.action = this.authConfigService?.getCustomLoginFormEndpoint();
    }
```

It also applies to other places where `isNewAuthFlow` used.